service/src/main/java/bio/terra/workspace/service/folder/model/Folder.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.workspace.service.folder.model;
```

Comment:
Can you assert that id and name are set?

https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/ea107a2c31ed014e6c333517b4e39050cea838c3/service/src/main/java/bio/terra/workspace/service/workspace/model/Workspace.java?q=missingrequiredfield#L183

Maybe not id, since it's not passed in CreateFolderRequestBody

Reply:
changed to record